src/Components/WebAssembly/WebAssembly/src/Rendering/WebAssemblyRenderer.cs

```diff
@@ -31,6 +32,10 @@ public WebAssemblyRenderer(IServiceProvider serviceProvider, ILoggerFactory logg
         {
             // The WebAssembly renderer registers and unregisters itself with the static registry
             _webAssemblyRendererId = RendererRegistry.Add(this);
+
+            // Since this is where generic unhandled errors get routed, we want the category
+            // name to be something generic, not "WebAssemblyRenderer".
+            _logger = loggerFactory.CreateLogger(".NET");
```

Comment:
I feel that this is a bit too generic `.NET`. Maybe we can use something more specific, like `Blazor`.

Is the problem that we have multiple logging sources comming from different places? What do we do on server-side Blazor. Can we be consistent with that?

Reply:
On Blazor Server, we do this:

https://github.com/dotnet/aspnetcore/blob/bc6bc2d22c42369627b5941e7c1dd98c4c8ad20a/src/Components/Server/src/Circuits/CircuitFactory.cs#L67

It makes a bit more sense for Blazor Server because you potentially have a lot of different things going on within your server-side ASP.NET Core process, so differentiating things from a circuit versus some other form of request handling is useful.

It doesn't make so much sense for Blazor WebAssembly because our programming model doesn't include other ways of running code.

However on balance I'm now siding with your consistency point, and will change this to `loggerFactory.CreateLogger<WebAssemblyRenderer>()`. It is a *bit* weird for that name to be what you have to use to filter these kinds of error messages, but theoretically you might run other code outside the Blazor rendering process, in which case it is a relevant differentiator.